patches/@graphiql+react+0.13.1.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/node_modules/@graphiql/react/dist/index.es.js b/node_modules/@graphiql/react/dist/index.es.js
+index f7ba535..752285c 100644
+--- a/node_modules/@graphiql/react/dist/index.es.js
++++ b/node_modules/@graphiql/react/dist/index.es.js
+@@ -18839,11 +18839,9 @@ function ExecutionContextProvider(props2) {
+               ...rest
+             };
+           }
+-          setIsFetching(false);
```

Comment:
This seems a bug in GraphiQL.
So `stop` button doesn't remain after the first result of a subscription.